lib/internal/validators.js

```diff
@@ -371,6 +371,8 @@ function validateEncoding(data, encoding) {
     throw new ERR_INVALID_ARG_VALUE('encoding', encoding,
                                     `is invalid for data of length ${length}`);
   }
+
+  return normalizedEncoding;
```

Comment:
What's the reason for this change?

Reply:
I need to analyse the result, bcz if result is undefined - it means that provided value was invalid

Reply:
You're not actually using this validator so this change can be removed.